unittests/test_utility.py

```diff
@@ -280,6 +280,22 @@ def test_load_unknown_path(self):
             self.assertEqual('foo', e.name)
             self.assertEqual('/foo', e.path)
 
+    def test_load_dir_starts_with_ddot(self):
+        with os_ext.change_dir('reframe'):
+            filename = '..%s/reframe' % os.getcwd()
```

Comment:
I don't think that this does the intended thing, because `os.getcwd()` returns an absolute path. I don't think it's needed. In any case the `change_dir()` assumes that we are running the unit tests from top-level directory.